x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/privilege_form_calculator/privilege_form_calculator.ts

```diff
@@ -272,7 +272,18 @@ export class PrivilegeFormCalculator {
 
     return feature.getPrimaryFeaturePrivileges().find(fp => {
       const correspondingMinimalPrivilegeId = fp.getMinimalPrivilegeId();
-      const hasMinimalPrivileges = feature.subFeatures.length > 0;
+
+      const correspendingMinimalPrivilege = feature
+        .getMinimalFeaturePrivileges()
+        .find(mp => mp.id === correspondingMinimalPrivilegeId)!;
+
+      // There are two cases where the minimal privileges aren't available:
+      // 1. The feature has no registered sub-features
+      // 2. Sub-feature privileges cannot be customized. When this is the case, the minimal privileges aren't registered with ES,
+      // so they end up represented in the UI as an empty privilege. Empty privileges cannot be granted other privileges, so if we
+      // encounter a minimal privilege that isn't granted by it's correspending primary, then we know we've encountered this scenario.
+      const hasMinimalPrivileges =
```

Comment:
Do we need to add a unit test to exercise case 2?

Reply:
This is exercised in the consuming component test suite here: 
https://github.com/legrego/kibana/blob/e266009dea156af0710cd76bc5179b81f4a6a197/x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/feature_table.test.tsx#L700-L732